src/Components/AddToDo.jsx

```diff
@@ -1,19 +1,25 @@
+import { observer } from "mobx-react";
 import React, { useState } from "react";
 import "./styles.css";
 
-const AddToDo = () => {
-  const [list, setList] = useState([]);
+const AddToDo = observer (({myProp}) => {
```

Comment:
`myProp` is not a good name always say `props`

Reply:
Understand